Bottom: afaa301014f70ec3ea85c62833b659a67fd61378
Top:    10985ea34bc77f492af8c08a058af1937d82a33b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 19:50:11 +0000

Refresh of nearly-working

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c947540..9a8c17f 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2397,12 +2397,6 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
       M_POLYAREA_separate_isected (&e, res, &holes, &a_isected);
       M_POLYAREA_label_separated (a_isected, b, FALSE);
       M_POLYAREA_Collect_separated (&e, a_isected, res, &holes, action, FALSE);
-      /* delete of a_isected which is left */
-      while ((p = a_isected) != NULL)
-      {
-        a_isected = p->next;
-        poly_DelContour (&p);
-      }
       M_B_AREA_Collect (&e, b, res, &holes, action);
       poly_Free (&b);
